pkg/controller/servicebindingrequest/nested/acc/acc.go

```diff
@@ -0,0 +1,46 @@
+package acc
```

Comment:
acc here stands for?

Reply:
It stands for "accumulator".

Reply:
I would suggest renaming the package to "accumulator" to avoid ambiguity.

Reply:
Solved in fd42fb4d2d4483e77569ed58152b6530a95a9a52.